eox_nelp/pearson_vue/api/v1/views.py

```diff
@@ -124,11 +126,12 @@ def create(self, request, *args, **kwargs):
         Returns:
             Response: Response object with status code 201 and an empty dictionary.
         """
-        serializer = self.get_serializer(data=request.data)
-        serializer.is_valid(raise_exception=True)
-        self.perform_create(serializer)
-        headers = self.get_success_headers(serializer.data)
-        return Response({}, status=status.HTTP_200_OK, headers=headers)
+        response = super().create(request, *args, **kwargs)
+
+        if response.status_code == status.HTTP_201_CREATED:
```

Comment:
This is so crazy, chaging normal status codes. But we depends on client logic 201 ->200